seekret-rules/aws.rule

```diff
@@ -2,7 +2,7 @@ secret_access_key:
   match: (\"|')?(AWS|aws|Aws)?_?(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9/\+=]{40}(\"|')?
 
 access_key_id:
-  match: (\"|')?(AWS|aws|Aws)?_?(ACCESS|access|Access)?_?(KEY|key|Key)?_?(ID|id|Id)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
+  match: (\"|')?(AWS|aws|Aws)?_?(ACCESS|access|Access)?_?(KEY|key|Key)_?(ID|id|Id)?(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
 
 account_id:
-  match: (\"|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?(\"|')?\s*(:|=>|=)\s*(\"|')?[0-9]{4}\-?[0-9]{4}\-?[0-9]{4}(\"|')?
+  match: (\"|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?(\"|')?\s*(:|=>|=)\s*(\"|')?[0-9]{4}\-?[0-9]{4}\-?[0-9]{4}(\"|')?
```

Comment:
Let's revert the `?` additions here to enforce `KEY` and `ID`. I think we may also want to enforce the work `AWS` as well since that's what most of the documentation says to use.